libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -238,12 +269,13 @@ export class BotFrameworkAdapter extends BotAdapter {
     }
 
     /**
-     * Deletes an activity that was previously sent to a channel. It should be noted that not all
-     * channels support this feature.
+     * Deletes an activity that was previously sent to a channel.
      *
      * @remarks
-     * Calling `TurnContext.deleteActivity()` is the preferred way of deleting activities as that
-     * will ensure that any interested middleware has been notified.
+     * Calling `TurnContext.deleteActivity()` is the preferred way of deleting activities (rather than calling it directly from the adapter), as that
+     * will ensure that any interested middleware will be notified.
+     *
+     * Note: Not all chat channels support this method. Calling it on an unsupported channel may result in an error.
```

Comment:
>Note [](start = 7, length = 4)

There's a special syntax for doing notes.  I think it's [!TIP] followed by a line break and the tip text.

Reply:
I added a few of these! Thanks for the [!TIP]